src/shadowbox/server/main.ts

```diff
@@ -96,14 +96,15 @@ async function main() {
 
   logging.info('Starting...');
 
+  const localhostIp = '127.0.0.1';
   const prometheusPort = await portProvider.reserveFirstFreePort(9090);
   // Use 127.0.0.1 instead of localhost for Prometheus because it's resolving incorrectly for some users.
   // See https://github.com/Jigsaw-Code/outline-server/issues/341
-  const prometheusLocation = `127.0.0.1:${prometheusPort}`;
+  const prometheusLocation = `${localhostIp}:${prometheusPort}`;
```

Comment:
Could you just hardcode the 127.0.0.1 instead of using the new variable? It's a well know constant, so it's more readable. Otherwise I need to look for its definition.

Change it here and in the two other lines below.